apiSpec/transcriptSchema.json

```diff
@@ -74,11 +109,21 @@
                             "items": {
                                 "$ref": "#/$defs/G2Point"
                             }
+                        },
+                        "bls_signatures": {
+                            "type": "array",
+                            "items": {
+                                "oneOf": [
+                                    {"$ref": "#/$defs/G1Point"},
+                                    {"$ref": "#/$defs/emptyString"}
+                                ]
+                            }
                         }
                     },
                     "required": [
                         "runningProducts",
-                        "potPubkeys"
+                        "potPubkeys",
+                        "bls_signatures"
```

Comment:
This doesn't follow the same casing as `runningProducts` and `potPubkeys`. Can you change it to `blsSignatures` ?

Reply:
Ack, this makes sense. My bad copying over the Pythonic naming into the JSON.

Reply:
Ack, fixed in https://github.com/ethereum/kzg-ceremony-specs/pull/33/commits/22680924fe194e85ed00919744d13c90401c76f2